docs/cli/zetatool/overview.md

```diff
@@ -0,0 +1,30 @@
+# ZetaTool
+
+ZetaTool is a utility tool for the Zeta-Chain project. It provides a command to fetch the ballot/cctx identifier from the inbound hash
+
+## Installation
+
+To install ZetaTool, clone the repository and build the project:
+
+```sh
+git clone https://github.com/zeta-chain/node.git
+cd node/cmd/zetatool
+go build -o zetatool
+```
+
+Alternatively you can also use the target `make install-zetatool`
```

Comment:
Let's just put this instruction, people just use the make to build. No need to specify about cloning the repo.
Let's just put
```
make install-zetatool
```

Reply:
This tool would be used by people who might not be running nodes or have the repo clones already; I think having the clone and build option would be good.

Reply:
The tool will almost exclusively be used internally